frontend/tests/acceptance/edit-person-test.js

```diff
@@ -11,7 +11,7 @@ import { fillIn } from "@ember/test-helpers";
 module("Acceptance | edit person", function(hooks) {
   setupApplicationTest(hooks);
 
-  test("/people/:id edit person data", async function(assert) {
+  skip("/people/:id edit person data", async function(assert) {
```

Comment:
why do we skip this now, is this because of the pikadate bug?

Reply:
Then add a comment why it is skipped :wink: